app/controllers/world_locations_controller.rb

```diff
@@ -44,4 +56,15 @@ def load_world_location
     @world_location = WorldLocation.with_translations(I18n.locale).find(params[:id])
   end
 
+  def parent_taxon
+    {
+      title: @world_location.title,
+      description: "Accessing UK services from #{@world_location.name}, advice for travelling to the UK, and help with trading between the UK and #{@world_location.name}."
+    }
+  end
+
+  def accordion_content
+    YAML.load_file(Rails.root + "config/worldwide_publishing_taxonomy_ab_test_content.yml")[@world_location.slug]
```

Comment:
I think this could be memoized, since we're calling it twice in the controller method